tests/test_dmlrunner.py

```diff
@@ -15,15 +15,24 @@
 from tests.testing_utils        import make_split_job
  
 
-session_filepaths = set()
-# @pytest.fixture(scope="function")
-# def cleanup():
-#     yield
-#     for session_filepath in session_filepaths:
-#         if os.path.isdir(session_filepath):
-#             shutil.rmtree(session_filepath)
-#     assert len((os.listdir('tests/artifacts/runner_scheduler/mnist/transformed/'))) == 0, \
-#         "Cleanup failed!"
+@pytest.fixture(scope='session')
+def transformed_filepath():
+    return 'tests/artifacts/runner_scheduler/mnist'+'/transformed'
+
+@pytest.fixture(scope='session')
+def transformed_filepath_2():
+    return 'tests/artifacts/runner_scheduler/test'+'/transformed'
+
+@pytest.fixture(scope='session', autouse=True)
+def cleanup(transformed_filepath, transformed_filepath_2):
+    # Will be executed before the first test
+    yield
+    # Will be executed after the last test
+    if os.path.isdir(transformed_filepath):
+        shutil.rmtree(transformed_filepath)
+    if os.path.isdir(transformed_filepath_2):
+        shutil.rmtree(transformed_filepath_2)
```

Comment:
litty 

You can go ahead move the Trello Card to complete ig (or just delete since you resolved it in this PR)

Reply:
Moved to complete